src/components/[guild]/Requirements/components/RequirementDisplayComponent.tsx

```diff
@@ -22,7 +22,7 @@ const RequirementDisplayComponent = ({
   rightElement = <RequirementAccessIndicator />,
   ...rest
 }: Props) => {
-  if (requirement.type === "HIDDEN")
+  if (requirement.visibility === "HIDDEN")
```

Comment:
I wasn't sure about this change, let me know if you have a better idea! It's needed so we can display "Some secret requirements" in the requirements list.

Reply:
I'm honestly not sure why we checked `type` here. I saw, we even had `| "HIDDEN"` in our previous type. I'd say, if the functionality still works, it should be fine :ayy:

Reply:
Yeah, we actually assigned `HIDDEN` to the type somewhere else, I guess that was a "temporary" workaround :'D